spec/controllers/stories/bulk_create_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Stories::BulkCreateController, type: :controller do
+  let(:project) { create :project }
+  let(:story_1) { attributes_for :story, project: project }
+  let(:story_2) { attributes_for :story, project: project }
+  let(:post_params) do
+    {
+      project_id: project.slug,
+      stories: [story_1, story_2]
+    }
+  end
+
+  context 'create stories to the project' do
+    before do
+      headers = { 'x-api-key': ENV['EXPORT_API_TOKEN'] }
```

Comment:
I think this test depends on the presence of `ENV['EXPORT_API_TOKEN']` in `.env`, right? Let's avoid this dependence, we can set it like this:

```suggestion
      ENV['EXPORT_API_TOKEN'] = 'some_token'
      headers = { 'x-api-key': ENV['EXPORT_API_TOKEN'] }
```